pyhorn/client.py

```diff
@@ -121,13 +123,45 @@ def job(self, job_id):
         job_ = ServicesEndpoint.job(self, job_id)
         return ServiceJob(job_, self)
 
-    def get(self, path, params={}, extra_headers={}):
+    @handle_http_exceptions()
+    def statistics(self):
+        statistics_ = ServicesEndpoint.statistics(self)
+        return ServiceStatistics(statistics_, self)
+
+    def get(self, path, params=None, extra_headers=None):
         headers = self.default_headers.copy()
-        headers.update(extra_headers)
+
+        if extra_headers is not None:
+            headers.update(extra_headers)
+
         url = urljoin(self.base_url, path)
         auth = HTTPDigestAuth(self.user, self.passwd)
-        resp = _session.get(url, params=params, headers=headers, auth=auth)
+        resp = _session.get(url,
+                            params=params,
```

Comment:
Yes, definitely. I'm not going to deal with it now since I'm not really in love with how the http stuff works anyway. The whole project is rife with non-DRY code waiting on a concentrated refactoring effort.